auth/auth.rb

```diff
@@ -0,0 +1,67 @@
+# Copyright 2017 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def implicit project_id:
+  # [START implicit]
+  # project_id = "Your Google Cloud project ID"
+
+  require "google/cloud/storage"
+
+  # If you don't specify credentials when constructing the client, the client
+  # library will look for credentials in the environment.
```

Comment:
Ohhh... the env var is on the page *right* before the code snippet, and it's `highlighted as monospace and easy to see` Cool deal 👍